OlxProject/pages/unsuccessful_search_page.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../tools/all_requires'
+
+class UnsuccessfulSearchPage
+  include TopSearchPart
+  include TopPart
+
+  NO_FOUND_MESSAGE = "Не найдено ни одного объявления"
+
+  def get_error_message
+    driver.find_element(xpath: "//div[contains(@class, 'emptynew  large lheight18')]/p").text
```

Comment:
isn't it better to make method error_message, then add method error_message_text and search_successful? In Your case I can't reuse this method anywhere, so if I want to add error_message_displayed? method, I will have to add error_message method.